consoleApp/src/main/java/com/infoshare/service/VolunteerAvailability.java

```diff
@@ -18,16 +23,22 @@ private void changeAvailability(Volunteer volunteer){
         String data = Util.readDataFromConsole(AVAILABILITY_HEADER, ValidatorEnum.YESNO);
         if (data.equalsIgnoreCase("Y")){
             volunteer.setAvailable(!volunteer.isAvailable());
+            System.out.println(AVAILABILITY_STATUS_UPDATE);
+            System.out.println(volunteer.isAvailable() ? STATUS_AVAILABLE : STATUS_UNAVAILABLE);
         }
     }
 
-    public void updateAvailability(){
-        // FileDb database = new FileDb();
-//        String email = Util.readDataFromConsole(GET_EMAIL, ValidatorEnum.EMAIL);
-//        Volunteer volunteerToUpdate = database.getVolunteer(email);
-//        if(volunteerToUpdate != null){
-//            changeAvailability(volunteerToUpdate);
-//            database.saveVolunteer(volunteerToUpdate);
-//        }
+    public void updateAvailability() {
+        try {
+            FileDb database = new FileDb();
+            String email = Util.readDataFromConsole(GET_EMAIL, ValidatorEnum.EMAIL);
+            Volunteer volunteerToUpdate = database.getVolunteer(email);
+            if(volunteerToUpdate != null){
+                changeAvailability(volunteerToUpdate);
+                database.saveVolunteer(volunteerToUpdate);
```

Comment:
Może to tylko u mnie, ale czy zmiany zapisywanie są na bieżąco czy tylko po zamknięciu programu?

Reply:
zmiany zapisują się po przejsciu wszystkich pytań w rejestracji, w momencie powrotu do menu - więc jeżeli o to chodziło jako o zamknięcie programu to tak